v3.1/Dockerfile

```diff
@@ -77,4 +87,8 @@ RUN apt-get update \
     && rm -r mapnik \
     && apt-get autoremove -y --purge $BUILD_DEPENDENCIES \
     && rm -rf /var/lib/apt/lists/* \
-    && ln -s /usr/local/lib/mapnik /usr/lib/mapnik
+    && ln -s /usr/local/lib/mapnik /usr/lib/mapnik \
+    && ln -s /usr/local/lib/mapnik /usr/local/lib/plugins 
+
+#WORKDIR /usr/local/lib
+#ENTRYPOINT ["mapnik-render"]
```

Comment:
You can remove these two lines

Reply:
mapnik-render requires to be run where it can access the plugins in "plugins/input" This is the reason for the symlink and the workdir.

Reply:
So it's related to this comment https://github.com/jawg/docker-mapnik3/issues/2#issuecomment-656823293

Mapnik team says:
> mapnik-render at this point is only designed to be used locally, from within a mapnik git clone. So it is not expected to work if installed.

So adding workarounds for these purposes is not necessary.